src/disco/sagas/disco.js

```diff
@@ -34,7 +34,10 @@ export function* fetchDiscoveryAddons({
 
     const addons = createExternalAddonMap({ results });
 
-    yield put(loadAddonResults({ addons }));
+    for (const addon of addons) {
+      yield put(loadAddon({ addon }));
```

Comment:
It's a bit sad to do this but we're going to remove this disco code soon, so I think it's fine.